src/styles/blocks/header.scss

```diff
@@ -0,0 +1,71 @@
+body {
+  margin: 0;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  padding: 0 50px;
+  align-items: center;
+  background-color: rgba(255, 255, 255, 1);
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    height: 40px;
+  }
+}
+
+.nav {
+  &__link {
+    color: #060b35;
+    text-decoration: none;
+    text-transform: uppercase;
+    font-weight: 500;
+    line-height: 60px;
+    height: 60px;
+    display: block;
+    transition: $transition-time;
+
+    &:hover {
+      color: #00acdc;
+    }
+  }
+
+  &__form {
+    list-style-type: none;
+    display: flex;
+    align-items: baseline;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__video {
+    margin-right: 0;
+    box-sizing: border-box;
+  }
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+.is-active::after {
+  content: '';
+  display: block;
+  height: 4px;
+  width: 37px;
+  background-color: #00acdc;
+  margin: 0 auto;
+  border-radius: 8px;
+  position: absolute;
+  top: 56px;
```

Comment:
The `top` property for the `::after` pseudo-element is set to `56px`. This might need adjustments for different screen sizes to ensure the underline is positioned correctly relative to the text. Consider using relative units or media queries if necessary.